n3fit/runcards/examples/nnpdf40-like.yml

```diff
@@ -91,16 +91,14 @@ dataset_inputs:
 
 ################################################################################
 datacuts:
-  t0pdfset: 240329-01-rs-nnpdf40like-baseline
+  t0pdfset: NNPDF40_nnlo_as_01180
   q2min: 3.49
   w2min: 12.5
 
 ################################################################################
 theory:
-  theoryid: 708
+  theoryid: 40000000
 
-sampling:
-  separate_multiplicative: false
```

Comment:
This is necessary to get the exact comparison of my comment but I believe that this should be the default now and I'll add it to this PR if it is fine for you 

@andreab1997 @RoyStegeman

(it should not be a big or relevant effect and I guess you are using it anyway, but it could have an effect in the closure tests)

Reply:
Yes, I believe that we should remove this key now, and just enforcing the correct behaviour.

Reply:
From this conversation I understood the default would be set to False such that it would allow us to remove these lines from the runcard, but instead just a comment was added

Reply:
I've added a comment in this PR and we take the default out in a different one (actually, @andreab1997 if you want to do the honors :P)